modules/applications/examples/ui-builder/production/docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+# Used for running a bonita env for production
+services:
+  bonita-ui-builder:
+    image: <my-uib-image-name>:<version>
+    container_name: bonita-ui-builder
+    environment:
+      BONITA_DEV_MODE: false
+      BONITA_API_URL: http://host.docker.internal:8080/bonita/API
+      APPSMITH_ENCRYPTION_PASSWORD: <encryption-password>
+      APPSMITH_ENCRYPTION_SALT: <encryption-salt>
```

Comment:
Are the password and salt really needed? If yes, we should say what is expected (any string of any length)

Reply:
no idea, I just transpose what was done in `docker run` command into a docker-compose file. If you know more than I do, feel free to precise the value you want to set here.

Reply:
Yes, there are mandatory env variable to encrypt and store some data in Appsmith.

Reply:
There is no indication for their value: https://docs.appsmith.com/getting-started/setup/environment-variables#appsmith_encryption_password

Reply:
C'est l'utilisateur qui définit ce qu'il veut non ? 
On a un partenaire qui m'a envoyé ça et ça a eu l'air de bien marcher : 
![image](https://github.com/user-attachments/assets/36f6f912-c3e1-4570-8ba1-f249e353b917)

Reply:
Il faudrait le préciser (qu'on peut mettre ce qu'on veut), pour éviter que les users se posent la question...

Reply:
https://documentation.bonitasoft.com/bonita/latest/applications/ui-builder/ui-builder-docker-installation